.github/workflows/tests.yml

```diff
@@ -14,6 +14,8 @@ jobs:
   unit-tests:
     name: "🛃 Unit tests"
     runs-on: ubuntu-20.04
+    container:
+      image: docker.io/library/fedora:latest
```

Comment:
Can we use https://quay.io/organization/fedora, or https://registry.fedoraproject.org/?

Reply:
Will do.  Thanks.